tests/unit/examples/test_notebooks_python_tf2.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sys
```

Comment:
I suggest you move the contents to `test_notebooks_gpu.py` file, since we have only three files in this folder.